### ISSUE TITLE
Linux npm install double-nesting import paths

### ISSUE THREAD

**Issue by EricCrosson** [typescript-tools/rust-implementation#4]
```
; npm i @typescript-tools/rust-implementation
npm ERR! code 1
npm ERR! path /home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation
npm ERR! command failed
npm ERR! command sh -c node npm/install.js
npm ERR! node:fs:1691
npm ERR!   handleErrorFromBinding(ctx);
npm ERR!   ^
npm ERR! 
npm ERR! Error: ENOENT: no such file or directory, link '/home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation/node_modules/binary-install/bin/monorepo' -> '/home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation/node_modules/.bin/monorepo'
npm ERR!     at Object.linkSync (node:fs:1691:3)
npm ERR!     at linkBinaryIntoBin (/home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation/npm/install.js:13:8) {
npm ERR!   errno: -2,
npm ERR!   syscall: 'link',
npm ERR!   code: 'ENOENT',
npm ERR!   path: '/home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation/node_modules/binary-install/bin/monorepo',
npm ERR!   dest: '/home/eric/workspace/scratch/try-install-typescript-tools-rust-implementation/node_modules/@typescript-tools/rust-implementation/node_modules/.bin/monorepo'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/eric/.npm/_logs/2021-11-23T13_16_19_299Z-debug.log
```

**Comment by EricCrosson** [comment]
Can we avoid symlinking by

- updating the path in our custom binary-install script to use rust-implementation/bin/_, and
- adding a `bin` property to the package.json?

**Comment by EricCrosson** [comment]
Fixed by d6281f5

Many thanks to:

- https://github.com/evanw/esbuild/releases/tag/v0.13.4
- https://github.com/evanw/esbuild/blob/95f10ac00359c6395ab0057fefc5dbe278b5db67/lib/npm/node-install.ts
- https://github.com/evanw/esbuild/blob/95f10ac00359c6395ab0057fefc5dbe278b5db67/lib/npm/node-platform.ts
- https://github.com/evanw/esbuild/commit/95f10ac00359c6395ab0057fefc5dbe278b5db67
- https://github.com/evanw/esbuild/blob/master/scripts/esbuild.js
